src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -155,7 +154,7 @@ pub type StateValueType = EncodedJoinRow;
 pub type HashValueType = JoinEntryState;
 
 type JoinHashMapInner<K> =
-    EvictableHashMap<K, HashValueType, PrecomputedBuildHasher, SharedStatsAlloc<Global>>;
+    EvictableHashMap<K, HashValueType, PrecomputedBuildHasher, TaskLocalAllocator>;
```

Comment:
The value itself might be collected from the state table which does not use the `TaskLocalAllocator`. Can we ensure it's accurate? 🤔

Reply:
We can’t, will refactor later